app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
The message 'Friends should buy {masks_to_buy} masks' is returned if any friend is not wearing a mask. Ensure this logic aligns with the task requirements, especially if all friends need to be checked before deciding on the message.